src/pages/SpacesDashboard/SpacesDashboard.tsx

```diff
@@ -112,6 +113,20 @@ export const SpacesDashboard: React.FC = () => {
             </div>
           </AdminTitleBar>
 
+          <div>
+            <ButtonNG
+              variant="secondary"
+              isLink
+              linkTo={generateUrl({
+                route: ADMIN_IA_WORLD_TOOLS,
+                required: ["worldSlug"],
+                params: { worldSlug },
+              })}
+            >
+              Tools
+            </ButtonNG>
+          </div>
```

Comment:
If this is the only button (not regarding typing a URL in browser's address bar) it can be wrapped in `<TesterRestricted>`

Reply:
was wrapped in the https://github.com/sparkletown/sparkle/pull/2747/commits/48e69fc02ffb5e7c698db5b8a414d473df6eb3c8